templates/static/accessibility.html

```diff
@@ -0,0 +1,138 @@
+
+  <p>{{ _("You can read the last full accessibility test report from 28 June 2019.") }}</p>
+
+  <p>{{ _("<a href='{url}' download>DAC Report - June 2019</a>").format(url = "https://00e9e64bac9206c97891d55b342ec14fcbd060bd86f5fcfd7a-apidata.googleusercontent.com/download/storage/v1/b/census-ci-craftcms/o/assets%2FONS-Census-Survey-2021-DAC-Accessibility-Report-June-2019.pdf.pdf?qk=AD5uMEtDN_n0XefOe88HcOOip9fwE_rTOJzdgHBiXX3yCjLzxfhasUlTdMnrUKXhaoFi65ZEh6AVlZ84RZiVg3yldO9vX8pTWV1wBI-01hhi8i6opHB6AbJFO-kKe2YAECZr8z6yxdyk53WLMohu-_ek0QRYZ4W6SwxLo9wLz-EybNJg1m9Mt2nUeW6TXLR0YA_-NGyqC0A6EodtMYEtyVTD_XvLijY0i0uKybP51yYf-FL8nS1skl5ZGK7U-Pc2I_Mg6TpMKyanFdiLDMHbWh8rKepXltl0WFbShkEzE2Jg9mXmhlZ2jRPUZRF_dhL2z-t9wUGCqcSZoO4lqRcfCA7aSlotZ-gKZ3bxtKT2Ldu-U6obP1dZ2uWYKWS4QZOMcJjFP6NpBx2_b15bF87QjuToFjQqYh1jJ4CdeXCdT_SsiS12T8AdFjEhp6J58Qdxv4zLCJug782Gr39y2d95s54ASi6hpKY0YFclwh-IxpJ-XKypID5gmw_ZJy6OYKRnsKkPw11DXsBACGTmxqP3awuhJvFcIFWzJepLV7anjNnII2t0OlrXMVsR3pVQGCKels3KIvIHllzg-yh_9aU51pRBaoEdxiLGgWvKnptQXAblsAEiiTCFSfQX-0mXIDErIgJnsBmsvjV9zkajaHa4xHZCNHJ3SYC67ATe3T2J3_rc-kuZAsi1Nfiyrf3FrH_LkAYDVY00lsnyodGFuHgX-it5S-jhqZk0Zmt01f2X0a_kRWGy2NPl8yb_hOWFH_8iVySv9mgLsiIK1KyXV5NYkU02sDuK5QvNWmNPf6ok2_VtYPy8jIJSECxxlExHUCBWXSiUKK4lzsqWGQTsg2Gr3qNT-0g-oVfwhoApJzkaBU7Z9ndz4FJYAiuJzunS_nt3-d33F0OH27Kz") }}</p>
+
+  <h2>{{ _("What we’re doing to improve accessibility") }}</h2>
+
+  <p>{{ _("We are continuing to work with various charities and local deaf communities to understand their needs for the census.") }}</p>
+
+  <p>{{ _("We are looking at designs to see if it is possible to have British Sign Language on every page. This is challenging as we have been unable to find another service that does this, but we continue to investigate.") }}</p>
+
+  <p>{{ _("We will act on feedback from the 2019 Rehearsal in October.") }}</p>
+
+  <p>{{ _("This statement was prepared on 11 September.") }}</p>
```

Comment:
Does it not need `It was last updated on 11 September.`?

Reply:
Ive checked this and its correct